src/table/table.component.tsx

```diff
@@ -186,27 +216,42 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
                                 {...otherHeaderGroupProps}
                                 {...provided.droppableProps}
                                 ref={provided.innerRef}
+                                sx={{
+                                  height: headerHeight, // Consistent height to account for headers that don't include any icons
+                                }}
                               >
                                 {headerGroup.headers.map((column, index) => {
                                   const { key, ...otherHeaderProps } =
                                     column.getHeaderProps();
 
+                                  const dataKey = column.render('id') as string;
```

Comment:
I changed this in my other PR but I can ask it here - why do we use `column.render('id')` and not just `column.id`?